lib/siwapp/invoices.ex

```diff
@@ -0,0 +1,70 @@
+defmodule Siwapp.Invoices do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "invoices" do
+    field :identification, :string
+    field :name, :string
+    field :email, :string
+    field :contact_person, :string
+    field :net_amount, :integer, default: 0
+    field :gross_amount, :integer, default: 0
```

Comment:
I would appreciate some order or comments on invoices fields. It's messy to read